doc/examples/cython/cython_examples/__init__.py

```diff
@@ -11,17 +17,8 @@
         compute_single_matrix_multiplication
 
 __all__ = [
-        "simple_func",
-        "fib",
-        "fib_int",
-        "fib_cpdef",
-        "fib_cdef",
-        "simple_class",
-        "masked_log",
-        "compute_self_corr_for_voxel_sel",
-        "compute_kernel_matrix",
-        "compute_single_self_corr_syrk",
-        "compute_single_self_corr_gemm",
-        "compute_corr_vectors",
-        "compute_single_matrix_multiplication"
-        ]
+    "simple_func", "fib", "fib_int", "fib_cpdef", "fib_cdef", "simple_class",
+    "masked_log", "compute_self_corr_for_voxel_sel", "compute_kernel_matrix",
+    "compute_single_self_corr_syrk", "compute_single_self_corr_gemm",
+    "compute_corr_vectors", "compute_single_matrix_multiplication"
+]
```

Comment:
Was this a linter change or a manual one? It seems nice but has the problem of making git merges harder. Not a big deal but I would've probably kept it as is.

Reply:
Yeah, this is a linter change. I also prefer the longer lines.